services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -6,6 +6,17 @@
     "Error": "S3Error",
     "Object": "S3Object"
   },
+  "shapeAdditions": {
+    "shapes":{
+      "PartType":{
+        "type":"string",
+        "enum":[
+          "DEFAULT",
+          "LAST"
```

Comment:
I am not sure we should have Enum called `DEFAULT` since Null also means Default here.

Example UploadPart with No/Null  PartType should be equal to UploadPart with DEFAULT.

Reply:
I think unset (null) and `DEFAULT` have different meanings and applications could have different logic to handle them. This also applies to all enums in other shapes.

Reply:
leaving as DEFAULT and LAST for now